### DISCUSSION TITLE
easytier隧道会自动在一条路径不可用的时候寻找路径吗

### DISCUSSION THREAD

**Discussion by assassinliujie** [EasyTier/EasyTier#244] [General]
今天配网遇到了个很抽象的事情
![image](https://github.com/user-attachments/assets/2ec43014-d6d3-4e05-98c4-9fd72ff9f10e)

WSL和C机通过TCP隧道连接到公网D机，并分别透传自己的子网网段
然后断开网卡2，会导致ABC没法访问互联网，但是两条tcp隧道不会断开，断开网卡1甚至都不会让母机无法上网，两条tcp隧道也不会断开
这是否是说easytier隧道在无法通过网卡2访问互联网时，自动选择了通过虚拟交换机--母机--网卡1的路径。还是说有其他什么原因才导致隧道没断开。测试能否访问互联网使用的是apt-get update,测试tcp隧道是否正常用了easytier-cli ping和通过192.168.99.0网段RDP访问母机这三种方式，均能成功
而且我测试过这个隧道没断开不是什么延迟bug，他还能保持rdp桌面连接，在搞不清原理的情况下我有点担心是不是我给网配混了

**Comment by assassinliujie** [comment]
192.168.99.0网段是由路由器分配的子网网段，不是hyperv交换机分配的

**Comment by KKRainbow** [comment]
这个有点神奇了。。可能跟 easytier 会尝试用所有网卡去发送数据有关系
